LEDs/shimmer_leds.c

```diff
@@ -482,15 +482,25 @@ void ShimLeds_blinkSetLwrErrorBluetooth(void)
 {
   if (ShimLeds_isBlinkTimerCnt200ms())
   {
-    Board_ledToggle(LED_LWR_YELLOW);
+#if defined(SHIMMER3) || defined(SHIMMER4_SDK)
+    Board_ledOn(LED_LWR_YELLOW);
+    Board_ledOff(LED_LWR_RED + LED_LWR_GREEN);
+#else
+    Board_ledLwrSetColourRgb(LED_PWM_OFF, LED_PWM_ON, LED_PWM_OFF); //Yellow
+#endif
   }
 }
 
 void ShimLeds_blinkSetLwrErrorSdCard(void)
 {
   if (ShimLeds_isBlinkTimerCnt200ms())
   {
-    Board_ledToggle(LED_LWR_GREEN);
+#if defined(SHIMMER3) || defined(SHIMMER4_SDK)
+    Board_ledOn(LED_LWR_GREEN);
+    Board_ledOff(LED_LWR_RED + LED_LWR_YELLOW);
+#else
+    Board_ledLwrSetColourRgb(LED_PWM_ON, LED_PWM_ON, LED_PWM_OFF); //Yellow
```

Comment:
as per earlier comment, code comment says yellow, we actually should be setting LWR green here and you're passing in R + G = yellow